### ISSUE TITLE
unrar: security update to 5.5.5

### ISSUE THREAD

**Issue by MingcongBai** [AOSC-Dev/aosc-os-abbs#670]
```
It was reported that unrar fixed a VMSF_DELTA memory corruption issue in
there latest version unrarsrc-5.5.5.tar.gz. This problem was reported to
Sophos AV in 2012 but never reach upstream rar.
```

Fixed CVE: CVE-2012-6706.

**Comment by MingcongBai** [comment]
Fixed with 7d004ff46c17084b1ab289c993ed69b57c1db99b.

**Comment by MingcongBai** [comment]
Use AOSA-2017-0045.
